### PR TITLE
Fix part of #1317: Fix lint issues in domain module in the kotlin files (exploration, onboarding, profile)

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fixes part of #1317
`exploration, onboarding, profile` package ktlint check

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by vinitamurthi — APPROVED** [review]

File: domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt [hunk 1]
```diff
@@ -609,6 +768,14 @@ class ProfileManagementController @Inject constructor(
...
     matrix.postRotate(rotate.toFloat())
-    return Bitmap.createBitmap(croppedBitmap, /* x= */ 0, /* y= */ 0, cropSize, cropSize, matrix, /* filter= */ true)
+    return Bitmap.createBitmap(
+      croppedBitmap, /* x= */
+      0, /* y= */
+      0,
+      cropSize,
+      cropSize,
+      matrix, /* filter= */
+      true
```
Comment (rt4914) [inline domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt]: Fix these inline comments here and everywhere else in this file.
```
      matrix,
      /* filter= */ true
```

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested nit change.

File: domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt (hunk 1 above)
Comment (anandwana001) [reply domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt]: Done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by anandwana001** [comment]
> LGTM, thanks.

Thank you for review and merge :)

**Pull request merged**